lib/install/management/appliance.go

```diff
@@ -652,5 +721,11 @@ func (d *Dispatcher) makeSureApplianceRuns(conf *config.VirtualContainerHostConf
 		return errors.New("timed out waiting for IP address information from appliance")
 	}
 
-	return fmt.Errorf("could not obtain IP address information from appliance: %s", updateErr)
+	// wait till the appliance is fully initialized
```

Comment:
I'd prefer it if this were called after `makeSureApplianceRuns` rather than here. If needed we can change the names to:
```
ensureApplianceInitializes
ensureComponentsInitialize
```

Calling it here means we've a large error handling block in the middle of the function rather than a clean separation between success and failure paths.